app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -126,4 +122,17 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return super.onCreateOptionsMenu(menu);
     }
 
+    public void createNotification() {
+
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTime(ISO8601Date.getTimeZoneDate(ISO8601Date.getDateObject(session.getStartTime())));
+        calendar.add(Calendar.MINUTE, -10);
```

Comment:
That was added in https://github.com/mananwason/open-event-android/commit/9b47e186867fb7baab1755eeaff1fa98795c8379